Misc/NEWS.d/next/Core and Builtins/2019-04-11-14-36-55.bpo-36588.wejLoC.rst

```diff
@@ -0,0 +1,5 @@
+On AIX, :attr:`sys.platform` doesn't contain the major version anymore.
+It is always ``'aix'``, instead of ``'aix3'`` .. ``'aix7'``.  Since
+older Python versions include the version number, it is recommended to
+always use the ``sys.platform.startswith('aix')``.
+Contributed by M. Felt in :issue:`36588`:.
```

Comment:
There's no need to include the issue number in the text of a news entry (this is mentioned explicitly in the UI of `blurb add`). The issue number is extracted from the filename and prepended to the text (see also https://pypi.org/project/blurb/).

So, I would suggest the following:
```suggestion
Contributed by M. Felt.
```